go/vt/topo/tablet.go

```diff
@@ -287,7 +289,9 @@ func (ts *Server) GetTabletsByCell(ctx context.Context, cellAlias string, opt *G
 		}
 		tablets = append(tablets, &TabletInfo{Tablet: tablet, version: listResults[n].Version})
 	}
-
+	slices.SortFunc(tablets, func(i, j *TabletInfo) int {
+		return cmp.Compare(i.Alias.Uid, j.Alias.Uid)
+	})
```

Comment:
Is this really just for tests (predictable and consistent results)? If so, we could order the results there in the test instead. Not that I think having predictable results here is a bad thing — I'm just not sure that it's helpful and warrants the cost at this layer. My gut feeling is that the caller should order the entire results that it gets for all tablets in all keyspaces and cells (whatever it's requesting) and then order the full set as it prefers (based on keyspace, cell, UID etc.).

Reply:
jinx 🤣

Reply:
Sounds good. It's moved to the test only now